Columbus/frontend/src/components/Notifications/Notification.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState , useEffect } from 'react'
+
+import { Box } from "@material-ui/core";
+import Typography from '@mui/material/Typography';
+import Avatar from '@mui/material/Avatar';
+
+export default function Notification(props) {
+
+    const { notification } = props;
+    const [message, setMessage] = useState(null);
+
+    useEffect(() => {
+        if(notification['actionType'] == 'follow') 
+        {
+            setMessage(notification['actionTaker'] + " followed you.")
+        }
+        else if(notification['actionType'] == 'like') 
+        {
+            setMessage(notification['actionTaker'] + " liked your post.")
+        }
+    }, [])
```

Comment:
Burada useEffect'in dependantı olarak notification['actionType'] vermek daha doğru olur gibi.

`
useEffect(() => {
         if(notification['actionType'] == 'follow') 
         {
             setMessage(notification['actionTaker'] + " followed you.")
         }
         else if(notification['actionType'] == 'like') 
         {
             setMessage(notification['actionTaker'] + " liked your post.")
         }
     }, [notification['actionType']])
`

Reply:
Bu aslında `notification['actionType']` her değiştiğinde tekrar useEffect içerisine girip conditionlara tekrar girmesi için